src/main/daml/Daml/Finance/Interface/Types/Date/DayCount.daml

```diff
@@ -12,8 +11,12 @@ data DayCountConventionEnum
   -- ^ Actual 360.
```

Comment:
What do you think about moving directly to the CDM-generated DayCountConvention enums?

Reply:
I was planning on doing it as a separate PR. It will probably require some additional daycount conventions as well, for example the ACT_ACT_AFB.